libmamba/src/core/activation.cpp

```diff
@@ -1215,4 +1215,97 @@ namespace mamba
 
         return out.str();
     }
+
+
+    std::string NuActivator::shell_extension()
+    {
+        return ".nu";
+    }
+
+    std::string NuActivator::shell()
+    {
+        return "nu";
+    }
+
+    std::string NuActivator::hook_preamble()
+    {
+        return "";
+    }
+
+    std::string NuActivator::hook_postamble()
+    {
+        return "";
+    }
+
+    fs::u8path NuActivator::hook_source_path()
+    {
+        return Context::instance().prefix_params.root_prefix / "etc" / "profile.d" / "micromamba.nu";
+    }
+
+    std::pair<std::string, std::string>
+    NuActivator::update_prompt(const std::string& /*conda_prompt_modifier*/)
+    {
+        return { "", "" };
+    }
+
+    std::string NuActivator::script(const EnvironmentTransform& env_transform)
+    {
+        std::stringstream out;
+        auto convert_to_nulist = [](std::string command)
+        {
+            /*Checks shell variable being set and converts shell lists to nushell lists if
+             possible otherwise returns the command as a string*/
+            std::string new_command = command;
+            std::size_t idx = 0;
+            bool add = false;
+            while ((idx = new_command.find(":", idx)) != std::string::npos)
+            {
+                new_command.replace(idx, 1, " ");
+                add = true;
+                idx++;
+            }
+            if (add)
+            {
+                new_command = fmt::format("[{}]", new_command);
+            }
+            return new_command;
+        };
+
+        if (!env_transform.export_path.empty())
+        {
+            out << "$env.PATH = " << fmt::format("[{}]", env_transform.export_path) << "\n";
```

Comment:
I was thinking more like

```suggestion
            out << fmt::format("$env.PATH = [{}]\n", env_transform.export_path);
```

Or even

```suggestion
            fmt::print(out, "$env.PATH = [{}]\n", env_transform.export_path);
```

if that works